Corona-Warn-App/src/main/java/de/rki/coronawarnapp/srs/ui/symptoms/calendar/SrsSymptomsCalendarViewModel.kt

```diff
@@ -32,6 +32,7 @@ class SrsSymptomsCalendarViewModel @AssistedInject constructor(
     val symptomStart = symptomStartInternal.asLiveData(context = dispatcherProvider.Default)
 
     val events = SingleLiveEvent<SrsSymptomsCalendarNavigation>()
+    val showLoadingIndicator = SingleLiveEvent<Unit>()
```

Comment:
This would work fine in successful submission ONLY, I think we need to use Boolean flag instead of Unit. If it failed 
showLoadingIndicator.postValue(false) so user can try again.

Reply:
Added a boolean flag.